src/server.ts

```diff
@@ -65,17 +65,25 @@ app.post('/', (req, res, next) => {
     chain: req.body.chain,
   }
 
-  // Try to find by address
+  // Try to find by address, return on success.
   try {
     const result = findByAddress(req.body.address, req.body.chain, repository);
-    res.status(200).send({ result })
+    res.status(200).send(result);
```

Comment:
I think this change might be wrong....but what is supposed to be sent here?

In the current version it looks like this?  
```
send( { [ { status: 'perfect', address: '0x....' } ] } )
```

Reply:
Reverted in 72ecb77